src/components/profile/Profile.js

```diff
@@ -52,7 +54,7 @@ Profile.navigationOptions = {
 const styles = StyleSheet.create({
   centered: {
     justifyContent: 'center',
```

Comment:
`justifyContent` is now a prop of `Section.Row`

Reply:
Done